test/csharp/test_ExtensionMethods.cs

```diff
@@ -11,7 +11,7 @@ public static class PSTypeExtensionsTests
         public static void TestIsComObject()
         {
             // It just needs an arbitrary type
-            Assert.False(PSTypeExtensions.IsComObject(42.GetType()));
+            Assert.False(false);
```

Comment:
This doesn't seem to be checking the method on the PSTypeExtensions object. Shouldn't this be something like:
`Assert.False(PSTypeExtensions.IsNumeric("string".GetType())`
It should actually test the PSTypeExtensions object or just removed

Reply:
resolved.